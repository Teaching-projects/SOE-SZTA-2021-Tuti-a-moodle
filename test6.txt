b
a
100
70
30
z
80
+
60
u
40