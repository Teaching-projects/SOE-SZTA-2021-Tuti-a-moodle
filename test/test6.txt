A
AL
b
a
100
70
30
2
B
BL
z
80
+
60
u
40
2
