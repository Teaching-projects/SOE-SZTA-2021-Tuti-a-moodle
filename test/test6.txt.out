Enter the name of the entity: Enter the lore of A: Enter the health of A: b is not a number. Try again.
Enter the health of A: a is not a number. Try again.
Enter the health of A: Enter the attack of A: Enter the defense of A: Enter the cooldown of A: 
Enter the name of the entity: Enter the lore of B: Enter the health of B: z is not a number. Try again.
Enter the health of B: Enter the attack of B: + is not a number. Try again.
Enter the attack of B: Enter the defense of B: u is not a number. Try again.
Enter the defense of B: Enter the cooldown of B: 
====================
A - HP: 100.0, DMG: 70.0, DEF: 30.0
B - HP: 80.0, DMG: 60.0, DEF: 40.0
====================
The Battle begins.
A and B clash!
====================
The Battle is over.
A won!